CONTRIBUTING.md

```diff
@@ -0,0 +1,158 @@
+- `refactor` (refactoring production code)
+- `test` (adding missing tests, refactoring tests; no production code change)
+- `chore` (updating grunt tasks etc; no production code change)
+
+and `<desc>` is a short description of the changes made.
+
+#### 3.3.2 Description
+Your description is where you can go into more detail about your changes.
+Please include _at least_ the following:
+- What you changed
+- Why you changed it
+
+If your PR is related to an issue, please include the issue number in the description.
+If it solves the issue, please include `Closes #<issue-number>` in the description. This will automatically close the issue when the PR is merged.
+
+#### 3.3.3 Labels
+Each PR MUST have at least one label. The following labels are available:
+- `bugfix` (bug fix) => PR's with this label must have a corresponding issue
+- `enhancement` (existing feature)
+- `documentation` (documentation)
+- `dependencies` (Updates a dependency) 
+- `feature` (new feature) 
+- `hotfix` (very small bugfix/typo fix)
+- `refactor` (refactoring production code)
+
+(PR's with no label will be rejected)
+
+#### 3.3.4 Content
+A PR may not contain code and commits not related to the issue it is solving or feature it is completing.
+F.ex. 
+- a PR solving issue #1 may not contain code or commits related to issue #2.
+- Do not include a commit that changes something in the docs when working on a discord command (unless the docs are related to the command)
+- Do not refactor code in cog A when working on cog B
+- ...
+
+
+## 4. Code Guidelines
+
+### 4.1 PEP8
+
+Please follow the PEP8 style guide. To do so, please use the [black](https://pypi.org/project/black/) python formatter.
+(All code in this repository is formatted using black)
+
+### 4.2 Documentation
+
+#### 4.2.1 Docstrings
+Please include docstrings for all functions and classes. These do not have to be long,
+but at least include the basic information (what the function does, what the parameters are, what the return value is, etc).
+
+#### 4.2.2 Comments
+Please include comments where necessary. This includes:
+- Comments explaining complex code
+- Comments explaining why you did something a certain way
+
+#### 4.2.3 Type Hints
+Please include type hints for all functions and classes.
+
+### 4.3 Complexity
+
+Please keep your code as simple as possible. We prefer longer, simpler code over shorter, more complex code.
+Avoid long list comprehensions, complex lambda functions, etc.
+
+### 4.4 Imports
+
+Please keep your imports as simple as possible. Avoid importing entire modules, and avoid importing multiple modules on one line.
+
+[!IMPORTANT]
+If you wish to use an external library, please ask first. We want to keep the number of external libraries to a minimum.
+(using external libraries already included in the requirements.txt is fine).
+
+[!WARNING]
+@sebastiaan-daniels must review external libraries for security vulnerabilities before they can be used.
```

Comment:
I think you meant to put these in block quotes?
```suggestion
> [!IMPORTANT]
> If you wish to use an external library, please ask first. We want to keep the number of external libraries to a minimum.
(using external libraries already included in the requirements.txt is fine).

> [!WARNING]
> @sebastiaan-daniels must review external libraries for security vulnerabilities before they can be used.
```

So that it will displayed like this:

> [!IMPORTANT]
> If you wish to use an external library, please ask first. We want to keep the number of external libraries to a minimum.
(using external libraries already included in the requirements.txt is fine).

> [!WARNING]
> @sebastiaan-daniels must review external libraries for security vulnerabilities before they can be used.

Reply:
Absolutely, thanks!